front-end/src/component/Driverlogin.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+class Driverlogin extends Component() {
```

Comment:
Hey why `Component()` here? Change this to functional component

Reply:
oh okay I hadn't noticed that actually deleted it I have started to work on it afresh